Anbox tend not to leave enough time for Android to start.
This patch gives more breathing room.
--- a/src/anbox/cmds/launch.cpp
+++ a/src/anbox/cmds/launch.cpp
@@ -35,7 +35,7 @@ namespace fs = boost::filesystem;
 
 namespace {
 constexpr unsigned int max_session_mgr_wait_attempts{10};
-const std::chrono::seconds session_mgr_wait_interval{5};
+const std::chrono::seconds session_mgr_wait_interval{25};
 constexpr unsigned int max_dbus_service_wait_attempts{10};
 const std::chrono::seconds dbus_service_wait_interval{5};
 
